api/routes/auth_test.go

```diff
@@ -185,7 +185,7 @@ func TestAuthUser(t *testing.T) {
 				Password: "testpassword",
 			},
 			requiredMocks: func() {
-				mock.On("AuthUser", gomock.Anything, gomock.Anything).Return(&models.UserAuthResponse{}, nil).Once()
+				mock.On("AuthUser", gomock.Anything, gomock.Anything, gomock.Anything).Return(&models.UserAuthResponse{}, nil).Once()
```

Comment:
According to [testify doc](https://pkg.go.dev/github.com/stretchr/testify/mock#Anything), the `gomock.Anything` must be used when the argument isn't relevant to the test made. I believe could be a good take to check if the arguments sent to the functions tested are really not necessary because it can cause a false positive on it.

Reply:
I mean, not only on this function, in general on tests modified.

Reply:
You could develop this idea when building the tests here: https://github.com/shellhub-io/shellhub/pull/3177/files#r1389292023